docs/go/workflows.md

```diff
@@ -42,4 +43,118 @@ A Workflow can have one or more such parameters.
+<!--SNIPEND-->
+
+## How to start a Workflow
+
+With the Go SDK, there are two ways that you can start a Workflow:
+
+1. Use the Go SDK `client` to start a Workflow from a Go process, as described below.
+2. Start a Workflow from an already running Workflow, which is known as a [Child Workflow](/docs/go/child-workflows).
+
+:::note
+
+Starting a Workflow is not the same as executing a Workflow.
+Starting a Workflow means that you are telling the Server to begin tracking the state of the Workflow execution.
+In a Temporal application, you do not run Workflow code directly, instead Workflow code is hosted and executed by a [Worker](/docs/go/workers).
+
+:::
+
+To start a Workflow you need to create the Temporal Go SDK client, call `ExecuteWorkflow()`, and pass it the following:
+
+1. `context.Context`, which is a normal [Go `Context` type](https://golang.org/pkg/context/).
+2. `client.StartWorkflowOptions` Which can include [timeout settings and a `RetryPolicy`](/docs/go/retries).
+3. The name of the Workflow function.
+4. Variables that should be passed to the Workflow when it begins execution.
+
+```go
+package main
+
+import (
+    "context"
+
+    "go.temporal.io/sdk/client"
+)
+
+func main() {
+    // Create the client object just once per process
+    c, err := client.NewClient(client.Options{})
+    if err != nil {
+        // Handle failure
+    }
+    defer c.Close()
+
+    // TaskQueue is the only required option to start a Workflow
+    options := client.StartWorkflowOptions{
+        TaskQueue: "your_task_queue",
+    }
+
+    we, err := c.ExecuteWorkflow(context.Background(), options, YourWorkflow, workflowParm1, workflowParam2)
+    if err != nil {
+        // Handle failure
+    }
+
+    // Use the WorkflowExecution to get the result
+    // Get is blocking call and will wait for the Workflow to complete
+    var workflowResult string
+    err = we.Get(context.Background(), &workflowResult)
+    if err != nil {
+        // Handle failure
+    }
+
+    // Do something with the result
+    printResults(workflowResult, we.GetID(), we.GetRunID())
+}
+```
+
+Starting Workflows synchronously or asynchronously is both possible.
+In Go, the only difference is whether the code waits for the result of the Workflow in the same process in which you started it.
+When you start a Workflow, a `WorkflowExecution` is returned.
+The `WorkflowExecution` can be used to get the result or capture the WorkflowId.
+Workflows do not rely on the process that invoked it, and will continue executing even if the waiting process crashes or stops.
+As long as you have the WorkflowID, which is available from the `WorkflowExecution` you can retrieve the result of the Workflow from a completely different process by using the `GetWorkflow` call on the client.
+
+```go
+we = client.GetWorkflow(workflowID)
+var result string
+we.Get(ctx, &result)
+```
+
+In most uses cases it is better to be prepared to execute the Workflow asynchronously.
```

Comment:
```suggestion
You can start workflows either **asynchronously** (most common) or **synchronously**.
In Go, the only difference is whether the code waits for the result of the Workflow in the same process in which you started it.

When you start a Workflow, a `WorkflowExecution` is returned.
The `WorkflowExecution` can be used to get the result (synchronous) or capture the WorkflowId (asynchronous).
Workflows do not rely on the process that invoked it, and will continue executing even if the waiting process crashes or stops.

As long as you have the WorkflowID, which is available from the `WorkflowExecution`, you can retrieve the result of the Workflow from a completely different process by using the `GetWorkflow` call on the client.

```go
we = client.GetWorkflow(workflowID)
var result string
we.Get(ctx, &result)
```

```

trying to break up the paragraph a little

Reply:
I hear ya.

Though, I don't think saying something is "most common" is actually a recommendation per se - more of an observation.

I will adopt most of your recommendations here.
However, I do have a strong preference towards laying out the state of things, and then providing a soft recommendation at the end. As the story preceding the recommendation lays the foundation for it.

wdyt about this?

![Screen Shot 2021-04-29 at 12 20 52 PM](https://user-images.githubusercontent.com/34380806/116599508-6f1bc480-a8e5-11eb-8a57-b89e66854f87.png)